packages/exhibitor/extra/dcos-exhibitor.service

```diff
@@ -35,3 +36,5 @@ ExecStartPre=$PKG_PATH/bin/bootstrap_exhibitor_tls.py
 ExecStartPre=$PKG_PATH/bin/set_exhibitor_file_permissions.py
 # Start Exhibitor
 ExecStart=$PKG_PATH/usr/exhibitor/start_exhibitor.py
+# Wait for ZooKeeper to start listening
+ExecStartPost=-timeout 20 /bin/sh -c 'until lsof -i :2181; do sleep 1; done'
```

Comment:
Looks like the test wants this to be `/usr/bin/timeout`

Reply:
I went ahead and changed the path and pushed to the branch will approve again once the tests pass again.